experimental/displaynames/src/displaynames.rs

```diff
@@ -5,12 +5,11 @@
 //! This module contains types and implementations for the Displaynames component.
 
 use crate::options::*;
-use crate::provider::LanguageDisplayNamesV1Marker;
+use crate::provider::LocaleDisplayNamesV1Marker;
```

Comment:
I acknowledge that this is probably a more correct name given that we have some locales that include script/region subtags. But, ECMA-402 has type: "language"

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames/DisplayNames

ICU4C has both languageDisplayName and localeDisplayName

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames/DisplayNames

(I don't have time at the moment to investigate the differences)

Reply:
So these are display names of _language identifiers_. For example, "zh-Hant-TW" is "Traditional Chinese (Taiwan)". That is indeed the name of a language.

Reply:
Thoughts on `LanguageIdentifierDisplayNamesV1Marker`? `LangIdDisplayNamesV1Marker`?

Reply:
I think LocaleDisplayNames is a misnomer.

I think LanguageIdentifierDisplayNames is a mouthful that doesn't add much value but it's fine.

I think LanguageDisplayNames is best because

1. The `of` function will soon take a LanguageIdentifier which signals fairly strongly that this is getting names of full language identifiers, not just language subtags
2. It's consistent with 402